src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -181,13 +180,22 @@ object Import {
   case class Endpoint(
     bindProtocol: String,
     bindPort: Int,
-    @deprecated("services are deprecated in favour of ACLs", since = "1.3.0") services: Option[Set[URI]],
+    services: Option[Set[URI]],
     serviceName: Option[String],
     acls: Option[Set[RequestAcl]])
 
+  object BundleConfVersions extends Enumeration {
+    val V_1_1_0 = Value("1.1.0")
```

Comment:
Shouldn't we name the versions `1.1` and `1.2` instead of `1.1.0` and `1.2.0`. We going to change the format only for major versions.